inst/templates/tidy_contributing.md

```diff
@@ -1,162 +1,41 @@
 # Contributing to {{{ package }}}
 
--   [Prerequisites](#prerequisites)
--   [Fixing typos](#fixing-typos)
--   [PR process](#pr-process)
-    -   [Fork, clone, branch](#fork-clone-branch)
-    -   [Check](#check)
-    -   [Style](#style)
-    -   [Document](#document)
-    -   [Test](#test)
-    -   [NEWS](#news)
-    -   [Re-check](#re-check)
-    -   [Commit](#commit)
-    -   [Push and pull](#push-and-pull)
-    -   [Review, revise, repeat](#review-revise-repeat)
--   [Resources](#resources)
--   [Code of Conduct](#code-of-conduct)
+This outlines how to propose a change to {{{ package }}}. For more detailed
+info about contributing to this, and other tidyverse packages, please see the
+[**development contributing guide**](https://tidyverse.org/contributing).
 
-This explains how to propose a change to {{{ package }}} via a pull request using
-Git and GitHub. 
-
-For more general info about contributing to the tidyverse, see the 
-[Resources](#resources) at the end of this document.
-
-## Prerequisites
-
-Before you do a pull request, you should always file an issue and make sure
-someone from the tidyverse team agrees that it’s a problem, and is happy with
-your basic proposal for fixing it. If you’ve found a bug, first create a minimal
-[reprex](https://www.tidyverse.org/help/#reprex).
-
-## Fixing typos
+### Fixing typos
 
 Small typos or grammatical errors in documentation may be edited directly using
-the GitHub web interface, so long as the changes are made in the _source_ file
-(see [Document](#document) for detail). If you do not know what this means, see
-the [Resources](#resources) section to learn more.
-
-## PR process
-
-### Fork, clone, branch
-
-The first thing you'll need to do is to [fork](https://help.github.com/articles/fork-a-repo/) 
-the [{{{ package }}} GitHub repo]({{{github_url}}}), and 
-then clone it locally. We recommend that you create a branch for each PR.
-
-### Check
-
-Before changing anything, make sure the package still passes `R CMD check`
-locally for you. When in doubt, compare your `R CMD check` results with current
-results for [{{{ package }}} on Travis]({{{travis_url}}}) (checks on Linux and/or 
-MacOS) and, if applicable, AppVeyor (checks on Windows). You'll do this again
-before you finalize your pull request, but this baseline will make it easier to
-pinpoint any problems introduced by your changes.
-
-``` r
-devtools::check()
-```
-
-### Style
-
-Match the existing code style. This means you should follow the tidyverse 
-[style guide](http://style.tidyverse.org). Use the 
-[styler](https://CRAN.R-project.org/package=styler) package to apply the style 
-guide automatically.
-
-
-Be careful to only make style changes to the code you are contributing. If you
-find that there is a lot of code that doesn't meet the style guide, it would be
-better to file an issue or a separate PR to fix that first.
-
-### Document
+the GitHub web interface, so long as the changes are made in the _source_ file.
```

Comment:
I'd add this

  * YES: you edit a roxygen comment in a `.R` file below `R/`
  * NO: you edit an `.Rd` file below `man/`